project-rome-docs/includes/android/notifications-dev-center-onboarding.md

```diff
@@ -9,8 +9,24 @@ ms.assetid: dc4d7bbd-bc87-42b1-9924-52c7bfcd5b5f
 ms.localizationpriority: medium
 ---
 
+### Register your app for push notifications
+
+Register your application with Google for [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging/android/client) support. Be sure to make note of the sender ID and server key that you receive; you'll need them later.
+
+Once registered, you must associate push notification functionality with the Connected Devices Platform in your app.
+
+```Java
+mNotificationRegistration = new ConnectedDevicesNotificationRegistration();
+            mNotificationRegistration.setType(ConnectedDevicesNotificationType.GCM);
+            mNotificationRegistration.setToken(token);
+            mNotificationRegistration.setAppId(Secrets.GCM_SENDER_ID);
```

Comment:
We should update this snippet to refer to FCM rather than GCM, like the rest of the text does.

Reply:
Also these lines should all use the same level of indentation.

---
In reply to: [251583060](https://github.com/MicrosoftDocs/project-rome/pull/33#discussion_r251583060) [](ancestors = 251583060)